Competition/src/main/cpp/Lift.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Lift.h"
+
+Lift::Lift() : liftMtrLeft{LiftConstants::VICTOR_ID_LEFT_Lift}, 
+             liftMtrRight{LiftConstants::VICTOR_ID_RIGHT_Lift},
+             liftServoLeft{LiftConstants::SERVO_ID_LEFT_Lift},
+             liftServoRight{LiftConstants::SERVO_ID_RIGHT_Lift},
+             operatorController(NULL) {
+    initLift();
+}
+
+void Lift::setController(frc::XboxController* controller) {
+    operatorController = controller;
+}
+
+void Lift::initLift() {
+    
+    liftMtrLeft.SetInverted(false);
+    liftMtrRight.SetInverted(false);
+}
+
+void Lift::setDefaultState() {
+    state.liftState = LiftState::DISABLED;
+
+    resetState();
```

Comment:
need to implement this function in the Lift class. Fill the function with setting the motor outputs in the state member struct to 0